src/azure-cli/azure/cli/command_modules/aro/tests/latest/test_aro_commands.py

```diff
@@ -10,17 +10,19 @@
 
 
 from knack.log import get_logger
+from azure.cli.command_modules.aro.tests.latest.custom_preparers import AROClusterServicePrincipalPreparer
 from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
 from azure.cli.testsdk.checkers import StringContainCheck
-from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk.scenario_tests import GeneralNameReplacer, AllowLargeResponse
```

Comment:
I think _GeneralNameReplacer_ is not being used (I could be wrong).

Reply:
You are correct, I'll fix this. Thanks!